processors/record_linker/processor.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import uuid
+import logging
+from .. import base
+logger = logging.getLogger(__name__)
+
+
+# Module API
+
+def process(conf, conn):
+
+    # Trial m2m
+    TRIAL_TABLES = [
+        'trials_conditions',
+        'trials_interventions',
+        'trials_locations',
+        'trials_organisations',
+        'trials_persons',
+        'trials_publications',
+    ]
+
+    # Unlink wrong linked records
+    # [will be reprocessed on next processors run]
+    count = 0
+    for record in base.helpers.iter_rows(conn, 'database', 'records', orderby='id'):
+        # Already unlinked
+        if not record['trial_id']:
+            continue
+        trial_records = list(conn['database']['records'].find(trial_id=record['trial_id']))
+        # Record not linked to others
+        if len(trial_records) == 1:
+            continue
+        linked_identifiers = set()
+        for trial_record in trial_records:
+            if record['id'] == uuid.UUID(trial_record['id']).hex:
```

Comment:
Because this `uuid` type in database is nightmare. Sometimes it's like `adsdfasdfds` sometimes like `sadfsda-sdafa-sdf` sometimes uuid object needing hex. I will never be using it again))) For uuid it could be plain text in db..